mongodb/Acornfile

```diff
@@ -0,0 +1,585 @@
+                }
+                if args.extraFlags != "" {
+                    "MONGODB_EXTRA_FLAGS": "\(args.extraFlags)"
+                }
+            }
+            if args.diagnosticMode {
+                command: ["sleep"]
+                args: ["infinity"]
+            }
+            if !args.diagnosticMode {
+                probes: [
+                    {
+                        type:                "liveness"
+                        initialDelaySeconds: 30
+                        periodSeconds:       20
+                        timeoutSeconds:      10
+                        successThreshold:    1
+                        failureThreshold:    6
+                        exec: command: [
+                            "bash",
+                            "/acorn/scripts/ping.sh"
+                        ]
+                    },
+                    {
+                        type:                "readiness"
+                        initialDelaySeconds: 5
+                        periodSeconds:       10
+                        timeoutSeconds:      5
+                        successThreshold:    1
+                        failureThreshold:    6
+                        exec: command: [
+                            "bash",
+                            "/acorn/scripts/ping.sh"
+                        ]
+                    }
+                ]
+            }
+            dirs: {
+                "/acorn/scripts": "./scripts"
+                "/bitnami/mongodb": "volume://mongodb-data-0"
+                if localData.tlsEnabled {
+                    "/certs/mongodb.pem": "secret://tls-certs/cert-key"
+                    "/certs/mongodb-cert": "secret://tls-certs/ca"
+                }
+            }
+        }
+    }
+    volumes: {
+        "mongodb-data-0": {}
+    }
+}
+
+// Replicaset instance
+if args.isReplicaSet {
+    for i in std.range(args.replicas) {
+        if localData.autoDiscovery && args.externalAccessEnabled {
+            jobs: {
+                "mongodb-service-discovery-\(i)": {
+                    image: "bitnami/kubectl"
+                    command: ["/acorn/scripts/service_discovery.sh"]
+                    dirs: {
+                        "/acorn/scripts": "./scripts"
+                        "/tmp/shared": "volume://mongodb-service-discovery-\(i)"
+                    }
+                    env: {
+                        "MY_POD_NAMESPACE": "\(args.appNamespace)"
+                        "MY_POD_NAME": "mongodb-\(i)"
+                    }
+                }
+            }
+            volumes: {
+                "mongodb-service-discovery-\(i)": {
+                    size: "10m"
+                }
+            }
+        }
+        containers: {
+            "mongodb-\(i)": {
+                image: "bitnami/mongodb:\(args.dbVersion)"
+                ports: {
+                    if args.externalAccessEnabled {

```

Comment:
I would always expose them. If the user wants to publish them they can do `acorn run -p 27017 [IMAGE]`.

Reply:
Ok, this arg was also used for endpoint auto-discovery job. But I think i will remove that part cause it has some issues. And it will just provide access inside the same k8s cluster